recipes/gclib/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
+
+build:
+    number: 0
+    skip: False
+
+source:
+  git_url: https://github.com/gpertea/gclib.git
```

Comment:
On github you can use this URL: `https://github.com/gpertea/gclib/archive/907a9d6484b92ea0c357baf2da5d85bc6e6b82b1.tar.gz`
+ a checksum and everything looks great!